package.json

```diff
@@ -30,22 +30,23 @@
   },
   "author": "Alex Iansiti <alex@pointapi.com>",
   "license": "MIT",
+  "dependencies": {
+    "socket.io-client": "2.3.0"
+  },
   "devDependencies": {
-    "@types/jest": "23.3.13",
+    "@types/jest": "24.0.18",
+    "@types/node": "^12.7.12",
```

Comment:
Is it fine for this requirement to have the "^"?

Reply:
It's something I overlooked. I'm updating it now